lib/generator.js

```diff
@@ -12,8 +12,10 @@ module.exports = function(locals) {
 
   if (Array.isArray(config.skip_render)) {
     skipRenderList.push(...config.skip_render);
```

Comment:
This doesn't need `config.skip_render.length > 0`, I can verify empty array doesn't get `push`-ed.